plutus-playground/README.md

```diff
@@ -0,0 +1,38 @@
+# Building
+
+## Server
+
+``` sh
+cd plutus-playground-server
+stack build
+stack exec -- plutus-playground-server psgenerator ../plutus-playground-client/src/
+stack exec -- plutus-playground-server webserver -p 8080 ../plutus-playground-client/dist/
```

Comment:
Can we get some `nix` instructions for those of us who use that?

Reply:
done